src/client/core/evaluator.ts

```diff
@@ -0,0 +1,49 @@
+import { typed, create, all, factory } from 'mathjs'
+import * as checker from './../../types/checker'
+
+const config = {}
+const factories = {
+  ...all,
+  // Override for string comparison
+  createEqual: factory(
+    'equal',
+    [],
+    () => (a: string | number, b: string | number) => {
+      return a === b
+    }
+  ),
+  createUnequal: factory(
+    'unequal',
+    [],
+    () => (a: string | number, b: string | number) => {
+      return a !== b
+    }
+  ),
+  // Custom if-else function
+  createIfElse: factory('ifelse', [], () =>
+    typed('ifelse', {
+      'boolean, string | number, string | number': (
+        condition: boolean,
+        a: string | number,
+        b: string | number
+      ) => (condition ? a : b),
+    })
+  ),
+}
+export const math = create(factories, config)
+
+export function operation(
```

Comment:
Could we rename this function to `evaluateOperation` because `operation` is a little misleading. This function uses the expression in the `Operation` object and proceeds to evaluate it

Reply:
Sounds fair - maybe @alexanderleegs can follow up with a PR

Reply:
I'll make the PR